CHANGELOG.md

```diff
@@ -0,0 +1,98 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [3.0.0.preview2] - [3.0.0.preview3]
+### Added
+- Add GetCallFlags
+- Add GasLeft syscall
+- Add Stateless to CallFlags
+- Add StackItem.ToJson
+- Add call __initialize automatically
+- Add vote for committee
+- Add secp256k1
+- Add relay-block-filter
+- Add a change list to optimize datacache commit
+- Add NotFound command
+- Add plugin Description field
+- Add base64 SYSCALLs
+- Add StrictUTF8
+- Add event name to System.Runtime.Notify
+- Add MaxBlockSystemFee
+- Add RIPEMD160 syscall
+- Add MPT
+- Create callback
+- Add EffectiveVoterTurnout
+- Add MaxVerificationGas
+
+### Changed
+- Add simplify access to storage for native contracts
+- Sync try-catch opcodes
+- Cache most recent block
+- Remove grouping
+- Optimize the GetPrice() mechanism of SYSCALLs
+- Decouple NativeContract from InteropService
+- Subscribe to RelayResult messages in ConsensusService
+- Modify iterators to public
+- Move cosigners to transaction attributes
+- Make NEP6Wallet.path public
+- Unify unhandled exceptions
+- Remove FromJson
+- Unify akka logs with neo logs
+- Update neo-vm package
```

Comment:
I think that we can avoid the update `neo-vm` we have multiple entries like this

Reply:
Have removed duplicate ones.